src/ray/object_manager/plasma/store.cc

```diff
@@ -286,10 +287,23 @@ uint8_t *PlasmaStore::AllocateMemory(size_t size, MEMFD_TYPE *fd, int64_t *map_s
     *error = PlasmaError::OK;
   }
 
+  // Trigger object spilling if current usage is above the specified threshold.
+  const int64_t footprint_limit = PlasmaAllocator::GetFootprintLimit();
+  if (footprint_limit != 0 && object_spilling_threshold_ < 1.0) {
+    const float allocated_percentage =
+        ((float)PlasmaAllocator::Allocated()) / footprint_limit;
+    if (allocated_percentage > object_spilling_threshold_) {
+      RAY_LOG(INFO) << "Triggering object spilling because current usage "
```

Comment:
```suggestion
      RAY_LOG(DEBUG) << "Triggering object spilling because current usage "
```
This message will show up a lot if the threshold is low. We already have INFO messages for spilling, so probably okay if we suppress these.